kubernetes.js

```diff
@@ -372,14 +372,37 @@ const createProject = async (project, options) => {
     const localService = await createService(project, options)
     const localIngress = await createIngress(project, options)
 
+    const k8sDelay = 1000
+    const k8sRetries = 10
```

Comment:
As these are duplicated lower down, I'm going to move them to top level variables so they are only set once and shared and also make them set from driver options

e.g.
```
this._k8sDelay = this._app.config.driver.options.k8sDelay || 1000
this._k8sRetries = this._app.config.driver.options.k8sRetries || 10
```